src/displayapp/screens/Adder.cpp

```diff
@@ -0,0 +1,340 @@
+#include "displayapp/DisplayApp.h"
+#include "displayapp/screens/Adder.h"
+#include <cstdlib>   // For std::rand
+#include <algorithm> // For std::max
+
+using namespace Pinetime::Applications::Screens;
+
+Adder::Adder(Pinetime::Components::LittleVgl& lvgl, Controllers::FS& fs) : lvgl(lvgl), filesystem(fs) {
+  InitializeGame();
+}
+
+Adder::~Adder() {
+  CleanUp();
+}
+
+void Adder::InitializeGame() {
+  LoadGame();
+
+  tileBuffer = new lv_color_t[TileSize * TileSize];
+  std::fill(tileBuffer, tileBuffer + TileSize * TileSize, LV_COLOR_WHITE);
+
+  displayHeight = LV_VER_RES;
+  displayWidth = LV_HOR_RES;
+
+  fieldHeight = displayHeight / TileSize - 2;
+  fieldWidth = displayWidth / TileSize - 1;
+  fieldOffsetHorizontal = (displayWidth - fieldWidth * TileSize) / 2;
+  fieldOffsetVertical = (displayHeight - fieldHeight * TileSize) / 2 + (TileSize + 0.5) / 2;
+
+  fieldSize = fieldWidth * fieldHeight;
+  field = new AdderField[fieldSize];
+
+  InitializeBody();
+  CreateLevel();
+
+  refreshTask = lv_task_create(
+    [](lv_task_t* task) {
+      auto* adder = static_cast<Adder*>(task->user_data);
+      adder->Refresh();
+    },
+    AdderDelayInterval,
+    LV_TASK_PRIO_MID,
+    this);
+
+  appReady = false;
+  vTaskDelay(20);
```

Comment:
in our case it doesn't make a difference, but I'd like to have the time in `ms`, not in ticks. This also makes us tick-time-independent

```suggestion
  vTaskDelay(pdMS_TO_TICKS(20));
```